dbms/src/Interpreters/ExpressionActions.cpp

```diff
@@ -349,6 +361,15 @@ void ExpressionAction::execute(Block & block) const
         break;
     }
 
+    case CONVERT_TO_NULLABLE:
+    {
+        // for expand usage, when original col is const non-null value, the inserted null value will break its const attribute in global scope.
+        block.getByName(col_need_to_nullable).column->convertToFullColumnIfConst();
```

Comment:
Why converted full column is not assigned to another column?

Reply:
make sense, addressed